src/rpc/blockchain.cpp

```diff
@@ -1004,6 +1004,13 @@ static UniValue pruneblockchain(const JSONRPCRequest& request)
 
     LOCK(cs_main);
 
+    // reject manual pruning in case if a blockfilterindex is syncing
+    ForEachBlockFilterIndex([](BlockFilterIndex& index) {
+        if (!index.IsSynced()) {
+            throw JSONRPCError(RPC_MISC_ERROR, "Cannot prune blocks because blockfilterindex is syncing.");
```

Comment:
bd11606a95022478e6ef1ccaada1589cff13a327

Do you see a way to test this?

Reply:
I'm currently working on making the block-/undo-filesize dynamic which would then allow testing this more reliable.

Reply:
Would be nicer to just check if the indexes are beyond the requested prune height...